src/client/v2/indexer/lookupAccountTransactions.ts

```diff
@@ -300,14 +300,12 @@ export default class LookupAccountTransactions extends JSONRequest {
    *        .currencyGreaterThan(minBalance - 1)
    *        .do();
    * ```
-   * @remarks
-   * If you are looking for transactions with the currency amount greater than 0, simply construct the query without `currencyGreaterThan` because it doesn't accept `-1`, and passing the `0` `currency-greater-than` value would exclude transactions with a 0 amount.
    *
    * @param greater
    * @category query
    */
   currencyGreaterThan(greater: number) {
-    this.query['currency-greater-than'] = greater;
+    this.query['currency-greater-than'] = greater.toString();
```

Comment:
nit: I'd add a comment for each of the client methods just in case to make it clear why we're converting to string for now:
```suggestion
    // We convert the following to a string for now to correctly include zero values in request parameters.
    this.query['currency-greater-than'] = greater.toString();
```